test/rss.cpp

```diff
@@ -166,3 +166,43 @@ TEST_CASE("W3CDTF parser behaves correctly") {
 		REQUIRE(rsspp::rss_parser::__w3cdtf_to_rfc822("") == "");
 	}
 }
+
+TEST_CASE("W3CDTF behaves correctly with different local timezones") {
```

Comment:
I must've missed that on the first review; sorry about that. I think this line should say "W3C DTF to RFC 822 conversion", not just "W3CDTF", because, well, time format doesn't have a behaviour—it's a static thing, right?